src/eos-updater-fetch.c

```diff
@@ -64,6 +64,35 @@ fetch_data_free (FetchData *data)
 
 G_DEFINE_AUTOPTR_CLEANUP_FUNC (FetchData, fetch_data_free)
 
+typedef struct {
+  MwscScheduleEntry *scheduled_entry;
+  GCancellable *general_cancellable;
+  GCancellable *scheduled_entry_cancellable;
```

Comment:
Please add a `  /* (owned) */` comment after each of these pointers, to document that they hold a strong reference. It’s not an official gobject-introspection annotation, but I hope to get round to proposing it as such someday. It’s useful documentation in any case.

Reply:
And the other ones which are owned, not just `scheduled_entry_cancellable`.

Reply:
Ah, okay, I thought you meant only the one that's fully owned by this struct (`scheduled_entry_cancellable`). But I see what you mean now.